src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/DocumentTag.java

```diff
@@ -59,7 +59,8 @@ public enum DocumentTag {
     INTERNAL_NON_STANDARD_DIRECTION_APPELLANT_LETTER("internalNonStandardDirectionAppellantLetter"),
     INTERNAL_NON_STANDARD_DIRECTION_RESPONDENT_LETTER("internalNonStandardDirectionRespondentLetter"),
     INTERNAL_HEARING_REQUIREMENTS_UPDATED_LETTER("internalHearingRequirementsUpdatedLetter"),
-  
+    INTERNAL_MAINTAIN_CASE_LINKS_LETTER("internalMaintainCaseLinksLetter"),
```

Comment:
Can "Maintain case links" represent multiple things? I know of 2, link appeal and unlink appeal I think. If there are different ways of maintaining case links, we should make this specific e.g. 'INTERNAL_UNLINK_APPEAL_LETTER'?
This will make it easier to differentiate between linking/unlinking and any other things we can do under maintain case links.

Reply:
confirmed with Elli. There will be scope in future to make event maintain case links responsible for unlinking an appeal as well as editing an appeal depending on user selection.
So we should distinguise maintain case links -> unlink appeal from the future scenario of maintain case links -> edit linked appeal